addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java

```diff
@@ -51,8 +53,9 @@
     /**
      * IP of vallox ventilation unit web interface.
      */
-    public ValloxMVHandler(@NonNull Thing thing) {
+    public ValloxMVHandler(@NonNullByDefault Thing thing, WebSocketClient webSocketClient) {
```

Comment:
This is not exactly what I mean. What I wanted to express with the term "on class level" is that the annotation should be added in front of the `class` keyword like in the `ValloxMVHandlerFactory`.

Reply:
Makes sense. As reasult I had to do some (maybe usefull) null checks as well.